features/step_definitions/consultation_steps.rb

```diff
@@ -21,6 +21,21 @@
   click_button "Save"
```

Comment:
This step definition can now be simplified to one line, reusing the new step definition you've created:

```ruby
step "I draft a new "English" language consultation \"#{title}\""
```

Reply:
good idea, thanks!